### PR TITLE
Add twir 352

### PR DESCRIPTION
**Pull request by nellshamrell**

### CODE REVIEW

File: _episodes/028-twir-352.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: "This Week in Rust - Issue 352"
+date: 2020-08-12T14:30:00Z
```
Comment (jonhoo) [inline _episodes/028-twir-352.md:3]: This date doesn't seem right?

Comment (nellshamrell) [reply _episodes/028-twir-352.md:3]: Oh! Fixing!

Comment (nellshamrell) [reply _episodes/028-twir-352.md]: Corrected - though I have back dated it to the 18th, as that's when I said it was published in the podcast

Comment (nellshamrell) [reply _episodes/028-twir-352.md]: Hope that is ok, if not I can change it.

Comment (jonhoo) [reply _episodes/028-twir-352.md]: Oh, yeah, that's fine.

**Pull request merged**